docs/docs/users/filecoin_services.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Filecoin Services
+hide_title: true
+sidebar_position: 6
+---
+
+<p align="center" style={{ display: 'flex' , justifyContent: 'space-around' }}>
+  <img src="/img/logo.png" alt="Forest logo"/>
+  <img src="/img/chainsafe_logo.png" alt="ChainSafe logo"/>
+  <img src="/img/filecoin_logo.png" alt="Filecoin logo"/>
+</p>
+
+This page provides an overview of the services and infrastructure provided by ChainSafe across the Filecoin ecosystem.
+
+:::tip
+You can check the status of many ChainSafe services at [status.chainsafe.dev](https://status.chainsafe.dev).
+:::
+
+## Forest Node
+
+Filecoin full node written in Rust aiming to provide a stable and performant implementation of the Filecoin protocol.
+
+<details>
+
+<summary>Forest repositories</summary>
+<p>
+Actively-maintained repositories that are part of the Forest project are:
+
+- [forest](https://github.com/ChainSafe/forest) - the central repository containing Forest node implementation, relevant tests and documentation.
+- [forest-iac](https://github.com/ChainSafe/forest-iac) - Infrastructure as Code for deploying Forest nodes, mirroring important Filecoin assets and other services supporting Forest development.
+- [fil-actor-states](https://github.com/ChainSafe/fil-actor-states) - state-only version of the [Filecoin actors](https://github.com/filecoin-project/builtin-actors), following semver versioning and providing a stable interface for Forest and other Filecoin implementations.
+</p>
+</details>
+
+## Forest Explorer Faucet
+
+Forest Explorer Faucet is a serverless application that allows users to request FIL on both mainnet and calibnet. The faucet is available at [forest-explorer.chainsafe.dev/faucet](https://forest-explorer.chainsafe.dev/faucet). The code is open source and available at [github.com/ChainSafe/forest-explorer](https://github.com/ChainSafe/forest-explorer).
+
+:::info
+Questions? Issues? Feedback? [Connect with the Forest team](./introduction.md#connect-with-us).
+:::
+
+## Bootstrap nodes
+
+Bootstrap nodes are essential for new peers joining the network. They provide a list of known peers to connect to, allowing the new peer to join the network quickly. ChainSafe provides several bootstrap nodes (both Forest and Lotus-based) for the Filecoin network (the `chainsafe.io` domain):
```

Comment:
They aren't on this domain, this also reads a bit funny which is why I'm suggesting to remove it
```suggestion
Bootstrap nodes are essential for new peers joining the network. They provide a list of known peers to connect to, allowing the new peer to join the network quickly. ChainSafe provides several bootstrap nodes (both Forest and Lotus-based) for the Filecoin network:
```